### DISCUSSION TITLE
Order of linear solvers

### DISCUSSION THREAD

**Discussion by karthichockalingam** [AMReX-Codes/amrex#3037] [Q&A] [answered]
Hello,

What is the order of the implicit (linear solvers) currently used in AMReX?
How high can you go? Is there an option to change order of the finite-difference scheme?

Many thanks!

**Comment by WeiqunZhang** [comment, marked as answer]
It's second order and second order only.  BoxLib, the predecessor of AMReX, did have 4th-order stencils.  (https://github.com/BoxLib-Codes/BoxLib/blob/master/Src/LinearSolvers/F_MG/cc_stencil_apply.f90#L194, https://github.com/BoxLib-Codes/BoxLib/tree/master/Src/LinearSolvers/C_CellMG4).  But they were not ported to AMReX.
